test/jestsetup.js

```diff
@@ -0,0 +1,10 @@
+import Enzyme, { shallow, render, mount } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+
+// React 16 Enzyme adapter
+Enzyme.configure({ adapter: new Adapter() });
+
+// Make Enzyme functions available in all test files without importing
+global.shallow = shallow;

```

Comment:
Don't create globals!
If you can import it in your test do it.

Reply:
Okay